db/model/roomType.js

```diff
@@ -123,19 +123,21 @@ module.exports = function roomType(seq, dataTypes) {
             resolve(inst);
           }
 
-          inst._previousDataValues.bots.forEach((botName) => {
-            seq.models.Bot.findOne({
-              where: {
-                name: {
-                  $iLike: botName,
+          if (inst._previousDataValues.bots) {
+            inst._previousDataValues.bots.forEach((botName) => {
```

Comment:
The work starting on line 127 and the work starting on line 142 will be occurring at the same time since you're not chaining them in a promise chain. Is that intentional? Is that safe?